packages/playwright-webkit/install.js

```diff
@@ -14,6 +14,13 @@
  * limitations under the License.
  */
 
-const { installBrowsersForNpmInstall } = require('playwright-core/lib/utils/registry');
+let installDefaultBrowsersForNpmInstall;
 
-installBrowsersForNpmInstall(['webkit']);
+try {
+  installDefaultBrowsersForNpmInstall = require('playwright-core/lib/utils/registry');
```

Comment:
This looks different to how it was before. Install... is a named export rather than a default export.

Reply:
Ouch, good catch!